data/data-pipeline/data_pipeline/etl/score/etl_score_post.py

```diff
@@ -207,6 +209,22 @@ def _create_tile_data(
             tiles_score_column_titles
         ].copy()
 
+        if drop_guam_usvi:
+            # Currently, we do not want USVI or Guam on the map, so this will drop all
+            # rows with the FIPS codes (first two digits of the census tract)
+            logger.info("Dropping USVI and Guam from tile data")
+            tracts_to_drop = []
+            for fips_code in constants.DROP_FIPS_CODES:
```

Comment:
If we go with Jorge's suggestion, if `constants.DROP_FIPS_CODES` is empty then this will do nothing, so don't even really need an additional check.